pkgs/os-specific/linux/unscd/remove-old-socket.diff

```diff
@@ -0,0 +1,133 @@
+diff --git a/nscd.c b/nscd.c
+index a71e474..6b7d807 100755
+--- a/nscd.c
++++ b/nscd.c
+@@ -154,6 +154,10 @@ vda.linux@googlemail.com
+ 
+ #define DEBUG_BUILD 1
+ 
++#define NSCD_NUM_SOCKS   1
++static const char *const nscd_socks[] = {
++	"/var/run/nscd/socket",
```

Comment:
`/var/run` is a symlink to `/run` these days. We shouldn't be using `/var/run` directly.